experiments/20161104-pol-assay-high-throughput/20161117-primer-design.md

```diff
@@ -0,0 +1,7 @@
+### Primers for sequencing entire PB2 sequence
+
+Forward: 20-mer primer that ends 100bp upstream of end of pCI sequence
+Reverse: 20-mer primer that ends 100bp downstream of start of pCI sequence
+
+Forward: GCACCTATTGGTCTTACTGA
+Reverse: TTTCACAAATAAAGCATTTT
```

Comment:
Can you annotate the predicted melting temperature of the primer? It should ideally be roughly in the 55-65ºC range. If not, extend the primer until you hit that range.